tn_api_pricing_booking_spec_nilou.yml

```diff
@@ -0,0 +1,874 @@
+                        type: array
+                        description: The segments chosen for the trip
+                        items:
+                            $ref: '#/components/schemas/segment'
+
+    segment:
+        type: object
+        description: The segment which is a part of an openjaw or a oneway
+        required:
+            - segment_id
+            - segment_type
+            - flights
+        properties:
+            segment_id:
+                type: string
+                description: Segment_id for the group
+                example: "14c9e2e5c95a5bf0f9f7712de24348e6aba88c32"
+            flights:
+                $ref: '#/components/schemas/flights'
+
+    flights:
+        type: object
+        description: A list of flights, each flight includes origin, destination, flight number, departure time and arrival time
+        required:
+            - origin
+            - departure_time
+            - flight_number
+            - destination
+            - booking_code
+            - carrier
+            - key
+        properties:
+            key:
+                type: string
+                description: Leg’s string identifier
+                required: true
+                example: "oHzo6pBAAA/B1d6hPLAAAA=="
+            origin:
+                type: string
+                description: IATA code of the flight origin
+                required: true
+                example: CDG
+            destination:
+                type: string
+                description: IATA code of the flight destination
+                required: true
+                example: YUL
+            booking_code:
+                type: string
+                description: The airline booking code that describes the flight cabin class
+                required: true
+                example: X
+            cabin_class:
+                type: string
+                description: Leg’s cabin class *Travelport customers only*
+                example: Economy
+            carrier:
+                type: string
+                description: Leg's carrier code
+                required: true
+                example: TS
+            flight_time:
+                type: integer
+                description: Flight duration in minutes
+                example: 475
+            flight_number:
+                type: string
+                description: Leg's flight number
+                required: true
+                example: 475
+            departure_time:
+                type: DateTime
+                format: 'yyyy-MM-ddTHH:mm:ssZ'
+                description: Departure time of the flight
+                required: true
+                example: "2019-04-07T13:15:00.000+02:00"
+            arrival_time:
+                type: DateTime
+                description: Arrival time of the flight
+                format: 'yyyy-MM-ddTHH:mm:ssZ'
+                example: "2019-04-07T15:10:00.000-04:00"
+            branding_identifier:
```

Comment:
```suggestion
            brand_identifier:
```

Reply:
Agreed - ill change it on MCFS to match this